src/uta/loading.py

```diff
@@ -46,9 +46,11 @@ def _get_cursor(con):
         return cur
 
     def align(s1, s2):
-        score, cigar = utaaa.needleman_wunsch_gotoh_align(s1.encode("ascii"),
-                                                          s2.encode("ascii"),
-                                                          extended_cigar=True)
+        import uta_align.align.algorithms as utaaa
```

Comment:
yes, this is intentional. The uta_align import is broken for me, due to the compilation issues with that library. As such this makes the import local, and we can load this file otherwise.

Reply:
Reece just provided a fix for uta-align. Added an upgrade to it to this PR.